Makefile

```diff
@@ -46,8 +46,8 @@ CUDA_DIR  ?= $(or $(patsubst %/,%,$(dir $(patsubst %/,%,$(dir \
 # export LSAN_OPTIONS=suppressions=.asanignore
 AFLAGS = -fsanitize=address #-fsanitize=undefined -fno-omit-frame-pointer
 
-OPT    = -O -g
-CFLAGS = -std=c99 $(OPT) -Wall -Wextra -Wno-unused-parameter -fPIC -MMD -MP
+OPT   ?= -O -g
+CFLAGS = -std=c99 $(OPT) -Wall -Wextra -Wno-unused-parameter -fPIC -MMD -MP -mavx
```

Comment:
dropped

Reply:
We have the current OPT example in the readme:

make OPT='-O3 -march=skylake-avx512 -ffp-contract=fast'

Reply:
Yes, and `-ffp-contract=fast` is important even when using GCC because `-std=c99` disallows it otherwise (the `-std=gnu*` variants allow contraction). That explicitly used `-march=skylake-avx512` to make a point, but `-march=native` is conceivable as a default.